readme.md

```diff
@@ -22,47 +22,47 @@ Requirements
 
 * Coldbox >= v4.5
 * Elasticsearch  >= v5.0
```

Comment:
Since the  Mapping Builder is only compatible with Elasticsearch >= v6.0, do we detect the version or do we add a caveat that the mapping builder only supports 6+?